geomstats/_backend/pytorch/__init__.py

```diff
@@ -767,3 +767,7 @@ def triu_to_vec(x, k=0):
     n = x.shape[-1]
     rows, cols = triu_indices(n, k=k)
     return x[..., rows, cols]
+
+
+def nan():
```

Comment:
Why did you add this function `nan`? I don't see it used in this PR. 

We should also avoid using `numpy`'s functions in tensorflow and pytorch backends: do tf and pytorch have the function that you are looking for?

Additionally, `numpy` does not seem to have a `nan` function, only a `nan` attribute:

<img width="661" alt="Screen Shot 2021-03-28 at 11 34 50 AM" src="https://user-images.githubusercontent.com/8267869/112761682-9cf5ac80-8fb9-11eb-897c-a01555660b01.png">

Reply:
I use ```nan``` in ```_geodesic_bvp```: when the process takes too long, it is terminated and returns an array full of ```nan```s. This attribute seems to be particular to numpy. An other possibility is to return an array full of zeros, but that doesn't seem very satisfactory either. What do you think ?

Reply:
Oh, got it! Sorry, I hadn't seen the line in `_geodesic_bvp`. In this case, do you think that we could use `math.nan` since the `nan` is used to populate an array anyway. This would avoid adding/maintaining it within the backends. Sorry to ask yet another change! :/ We're almost there though.

Reply:
Yes good idea to use ```math.nan``` instead, perfect ! Thanks !